libbeat/docs/upgrading.asciidoc

```diff
@@ -93,6 +77,45 @@ we've provided a migration tool to help you migrate your configurations. For
 more information, see the
 https://www.elastic.co/blog/beats-6-7-0-released[Beats 6.7.0 release blog].
 
+==== Upgrade {beats} binaries to 7.0
+
+// REVIEWERS: I mirrored the steps for Elasticsearch because I figured they
+// would be similar, but I have not tried these steps on all platforms. Please
+// confirm that they are accurate.
+
+Before upgrading:
+
+. Back up the `data` and `config` directories. 
+. Stop the existing {beats} process by using the appropriate command for your
+system.
```

Comment:
Well, first stop the Beat. e.g. trying to backup data while Filebeat is writing the registry might not be optimal.

The data and config directories depend on the installation method. If Beats have been installed from an archive, then the location should be known to the user. Otherwise see: https://www.elastic.co/guide/en/beats/filebeat/master/directory-layout.html#_default_paths

If in doubt, start the Beat on the terminal. Config and data paths are printed at the beginning of Beats startup.

Reply:
yup, makes sense. I'll move the steps around and add more info about finding the location of data/config files.